examples/tensorboard_simple.py

```diff
@@ -0,0 +1,60 @@
+import optuna
+from optuna.integration.tensorboard import TensorBoardCallback
+from optuna import type_checking
+import tensorflow as tf
+
+if type_checking.TYPE_CHECKING:
+    from typing import Dict  # NOQA
+    from typing import Any  # NOQA
```

Comment:
In the new type hinting style, we can remove the `type_checking`, just import `typing/Dict or Any` outside `if type_checking.TYPE_CHECKING:` and remove NOQAs.